### ISSUE TITLE
Consider changing the version of nuget.exe hosted by www.nuget.org/nuget.exe to latest

### ISSUE THREAD

**Issue by scottbommarito** [NuGet/NuGetGallery#4559]
Currently, https://www.nuget.org/nuget.exe redirects to nuget.exe version 2.8.6. We should consider changing it to latest.

**Comment by anangaur** [comment]
IMO this should point to the latest and not the "latest recommended"

**Comment by scottbommarito** [comment]
@anangaur  I agree, but we're concerned that changing this may break clients who expect 2.8.6. I'm not sure how much the API has changed but we could be breaking a lot of people's CIs with this.

**Comment by anangaur** [comment]
@scottbommarito True. May be we can do a twitter poll to understand the usage?

**Comment by scottbommarito** [comment]
@anangaur I looked at our telemetry today, 13k users over the last day. (https://github.com/NuGet/NuGetGallery/issues/4558)

**Comment by joelverhagen** [comment]
I think we should leave it pointing to 2.8.6 and forget about it. If someone asks where to get nuget.exe we always say dist, which is the right answer.
